src/routes/transactions/constants.ts

```diff
@@ -4,3 +4,4 @@ export const ADDRESS_PARAMETER_TYPE = 'address';
 export const TRANSFER_PREFIX = 'transfer';
 export const MODULE_TRANSACTION_PREFIX = 'module';
 export const MULTISIG_TRANSACTION_PREFIX = 'multisig';
+export const TRANSACTION_ID_SEPARATOR = '_';
```

Comment:
I decided to make this separator (`_`) a constant since is now used in several places and logics. In the tests, the raw `_` char is still used.